SpatialGDK/Documentation/content/glossary.md

```diff
@@ -83,7 +83,9 @@ You can define interest in three ways, which you can use alongside each other:
 
 ### Offloading
 
-Offloading is the new architecture that the SpatialOS GDK for Unreal provides to allocate the authority of specific [Actor groups]({{urlRoot}}/content/workers/offloading-concept#actor-groups) from the main Unreal server-worker instance to a different server-worker instance. By using offloading, you can save the resources of the main Unreal server-worker instance when you want to build richer game features.
+Offloading is one of the multiserver options for working with SpatialOS. It involves allocating the [authority](#authority) over specific [Actor groups](#actor-groups) to a [server-worker instance](#worker-types-and-worker-instances) other than your main Unreal server-worker instance. 
+
+For example, you could offload computationally expensive but latency-tolerant systems, such as advanced AI or large-scale background physics simulation, so that your main server-worker instance can run other systems at a larger scale.
```

Comment:
If we use the text for Overview, as suggested here, we could amend this slightly.
Also, standardising with "computation" as what worker instances do.

```suggestion
Advanced AI and large-scale background physics computation are good candidates for offloading as they are computationally expensive but latency-tolerant. This would leave your game's out-of-the-box main server-worker instance to run other game systems at a larger scale.
```